### PR TITLE
Add instructions for installation

### PR DESCRIPTION
**Pull request by xmnlab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,5 +156,60 @@ You have successfully installed the OmniSci Datascience Installer on your Mac in
...
+
+Now you can access the OmniSci Data Science stack using the new environment you just created!
 
 ## Using the OmniSci Mac Preview
+
+If you already have [Mac Preview](https://www.omnisci.com/mac-preview) and you want to connect to that from your jupyter-lab notebook, first open your jupyter-lab instance. From the command line, go to the folder you want as root for your notebooks (or use --notebook-dir with the path of your desired folder) and call the `jupyter-lab` command, for example:
+
+```sh
+# $HOME is a variable that points to the path of you user folder
+mkdir $HOME/notebooks
```
Comment (scopatz) [inline README.md:175]: `mkdir -p $HOME/notebooks`

**Comment by scopatz** [comment]
Looks mostly good!

File: README.md (hunk 1 above)
Comment (xmnlab) [reply README.md:175]: thanks @scopatz, I am going to fix that now!

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**